cloud/functions/src/crawlers/crawl.ts

```diff
@@ -0,0 +1,58 @@
+import * as _ from "lodash";
+
+import {db, info} from "../firebase"
+import {crawl as crawlDevoxx} from "./devoxx/crawler"
+import { FullEvent } from "../models/Event";
+
+const crawlAll = async function() {
+    info("Starting crawling");
+
+    const events = []
+
+    const snapshot = await db.collection("crawlers/devoxx/events").where("crawl", "==", true).get();
+    if (snapshot.empty) {
+        info("no events to crawl")
+    } else {
+        for (const doc of snapshot.docs) {
+            info("crawling devoxx event " + doc.id)
+            const event = await crawlDevoxx(doc.id)
+            await saveEvent(event)
+            events.push({id: event.id})
+        }
```

Comment:
```suggestion
        await Promise.all(snapshot.docs.map(async doc => {
            info("crawling devoxx event " + doc.id)
            const event = await crawlDevoxx(doc.id)
            await saveEvent(event)
            events.push({id: event.id})
        }))
```
Let's parallelize crawlings (went from 29s to 19s on my laptop)